Library/Homebrew/brew.sh

```diff
@@ -262,22 +262,6 @@ check-array-membership() {
   fi
 }
 
-# Let user know we're still updating Homebrew if brew update --auto-update
-# exceeds 3 seconds.
-auto-update-timer() {
-  sleep 3
```

Comment:
Not a strong opinion, but feels nicer to try tweaking the 3 second timer before jumping to showing it all the time.

Reply:
@carlocab I think having it (almost) immediately print output makes the output more responsive (and, in turn, Homebrew "feel" faster).

Reply:
Trying this out I agree that having the immediate feedback feels nicer. Otherwise I wonder whether brew has started updating, or whether it's just blocked on loading files or something.